src/modules/LanItems.rb

```diff
@@ -2460,6 +2349,40 @@ def drop_hosts(ip)
       Host.set_names(ip, [])
     end
 
+    #  Creates a list of udev rules for old style named interfaces
+    #
+    #  It takes a whole "interfaces" section of AY profile. Keeps
+    #  only old style named interfaces (e.g. eth-bus-0.0.0000) in it
+    #  and renames them to a new name (e.g. eth0). It also produces
+    #  a list of udev rules to guarantee device naming persistency.
+    #  The rule is base on attributes described in old style name
+    #
+    #  @param [Array] list of hashes describing interfaces in AY profile
+    #  @return [Array] list of hashes for udev rules
+    def createUdevFromIfaceName(interfaces)
```

Comment:
same as bellow

I was also thinking about changing name and creating an alias for old one ... But didn't do so - it seems to be useless when it was moved elsewhere anyway